app/Repositories/SignupRepository.php

```diff
@@ -37,28 +29,17 @@ public function create($data)
         // Let Laravel take care of the timestamps unless they are specified in the request
         // @TODO: keep only the else after the migration
         if (isset($data['created_at'])) {
-            // Associate the event with the signup
-            $event->signup_id = $signup->id;
-
             // Set the created_at time
             $signup->created_at = $data['created_at'];
-            $event->created_at = $data['created_at'];
 
             // Set the updated time if provided, if not, assume no updates
             if (isset($data['updated_at'])) {
                 $signup->updated_at = $data['updated_at'];
-                $event->updated_at = $data['updated_at'];
             } else {
                 $signup->updated_at = $data['created_at'];
-                $event->updated_at = $data['created_at'];
             }
 
             $signup->save(['timestamps' => false]);
```

Comment:
When we run the migration, this will create a signup event when the signup is created (I think?) and then create another event when we save it here to change the timestamps, which I don't think we want because it's not really a real event. I couldn't figure out how to create a model without timestamps, only to save it. The workaround that immediately comes to mind is to change the event trigger from saved to created until we are done with the migration but I'm for sure up for discussion.